app/public/javascripts/timer.js

```diff
@@ -0,0 +1,95 @@
+class Timer {
+    constructor(root) {
+      root.innerHTML = Timer.getHTML();
+  
+      this.el = {
+        minutes: root.querySelector(".timer__part--minutes"),
+        seconds: root.querySelector(".timer__part--seconds"),
+        control: root.querySelector(".timer__btn--control"),
+        reset: root.querySelector(".timer__btn--reset")
+      };
+  
+      this.interval = null;
+      this.remainingSeconds = 120;
+  
+      this.el.control.addEventListener("click", () => {
+        if (this.interval === null) {
+          this.start();
+        } else {
+          this.stop();
+        }
+      });
+  
+      this.el.reset.addEventListener("click", () => {
+        const inputMinutes = prompt("Enter number of minutes:");
+  
+        if (inputMinutes < 60) {
+          this.stop();
+          this.remainingSeconds = inputMinutes * 60;
+          this.updateInterfaceTime();
+        }
+      });
+    }
+  
+    updateInterfaceTime() {
+      const minutes = Math.floor(this.remainingSeconds / 60);
+      const seconds = this.remainingSeconds % 60;
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 4 spaces but found 6.](https://app.codacy.com/gh/cse110-fa21-group17/cse110-fa21-group17/pullRequest?prid=8445086)
```suggestion
    const seconds = this.remainingSeconds % 60;
```